### ISSUE TITLE
[Bug] Lights, Glow Tiles, Spotlights etc can have wrong colour when spawning in BPs

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#5241]
### Short Description of the issue:
Sometimes when you spawn in a blueprint, the colour of emissive bricks on the build can display wrong. If you use the MMB, it shows the colour of the actual brick is still correct, but the light it casts is incorrect. The incorrect light cast is usually a very unusual colour like green or purple, and very bright.

This can happen on any blueprint, but I'll show some examples on this specific one.

This Glow Pipe in the cockpit should be blue, but when this BP was spawned it shows as vibrant purple. The 1st glow tile in the sequence of 3 (outlined in red) should also be blue, but spawns in as black
![image](https://github.com/tsunamayo/Starship-EVO/assets/27857711/99acff3f-90d4-4986-95c5-ce72756d5bac)
Sometimes the three glow tiles on the right can show the incorrect colours too.

This ship has spotlights that mimic thruster plumes. The spotlight on the left spawned in with a purple colour, whereas the one on the right is off (as it should be when not piloting the ship, it's hooked up with an Event Gate).
![image](https://github.com/tsunamayo/Starship-EVO/assets/27857711/306bf69e-8b0f-45bd-9e8f-b510d42c652a)

### Steps to reproduce the issue:
- Spawn the attached BP. You may need to spawn several to get the bug to show.
- Check the cockpit lights, they should show like below - 
- Also check the plume thrusters
![Y-Wing Cockpit](https://github.com/tsunamayo/Starship-EVO/assets/27857711/12c73d26-0964-41e6-99b7-41bc2361371f)

### Environment
- Starship EVO version: 23w33a
- Graphics Card: RX 6700XT

**Comment by CoasterBlock** [comment]
Same here! I encountered more behavior as well, the glow-bricks can have a "null-color" that will appear like a wierd magenta color, or black like a black hole lol. I reported it as issue #5071

**Comment by Tackietacktack** [comment]
Yes, I have it sometimes, if I have build too much light or if I have connected too much things(>20) to one logic gate. Could it be, that I have not enough ram memory ?
